index.html

```diff
@@ -0,0 +1,1410 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Newsweek</title>
+    <script src="https://kit.fontawesome.com/02aeb58d23.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <!-- Navbar -->
+
+      <nav class="navbar nav1 sticky-top navbar-dark font-weight-bold py-1">
+        <a class="navbar-brand" href="#">Newsweek</a>
+        <form class="form-inline">
+          <a class="mr-2">LOGIN</a>
+          <button class="px-2 border-0 font-weight-bold" type="submit">SUBSCRIBE ></button>
+          <button type="submit" class="navbar-toggler-icon font-weight-bold border-0 ml-2 d-inline-block mr-0 p-3"></button>
+        </form>
+      </nav>
+
+      <nav class="navbar nav2 container navbar-dark font-weight-bold my-0 py-0 mx-auto align-text-bottom">
+        <form class="form-inline">
+          <div>
+            <a href=""><i class="fas fa-cloud"></i> Alexandria, EG</a>
```

Comment:
Add the description in all `href` attribute of index page

Reply:
Done